### DISCUSSION TITLE
shizuku在无线调试配对成功后启动问题

### DISCUSSION THREAD

**Discussion by gzz4454** [RikkaApps/Shizuku#288] [Q&A]
无线调试配对已成功，但是点击启动后还是提示正在搜索无线调试服务，请问是什么原因

**Comment by gzz4454** [comment]
手机是pixel7 pro

**Comment by haruue** [comment]
先在开发者选项里禁用并重新启用无线调试， 然后重试启动 Shizuku Server。

**Reply by gzz4454** [reply]
试过了，并没有效果

**Comment by thgifhtgf** [comment]
> 先在开发者选项里禁用并重新启用无线调试， 然后重试启动 Shizuku Server。

系统更新后遇到一样问题，十次中一次成功启动但是权限是假的用不了

**Reply by thgifhtgf** [reply]
联想zui 15.0.774(安卓13)
